src/main/java/com/powsybl/sld/server/SingleLineDiagramService.java

```diff
@@ -120,8 +121,8 @@ SvgAndMetadata generateSvgAndMetadata(UUID networkUuid, String variantId, String
             sldParameters.setSubstationLayoutFactory(substationLayoutFactory);
             sldParameters.setVoltageLevelLayoutFactoryCreator(voltageLevelLayoutFactory);
             sldParameters.setLayoutParameters(layoutParameters);
-            sldParameters.setStyleProviderFactory(n -> diagParams.isTopologicalColoring() ?
-                    new StyleProvidersList(new TopologicalStyleProvider(network), new HighlightLineStateStyleProvider(network)) :
+            sldParameters.setStyleProviderFactory((n, s) -> diagParams.isTopologicalColoring() ?
+                    new StyleProvidersList(new TopologicalStyleProvider(network, svgParameters), new HighlightLineStateStyleProvider(network)) :
```

Comment:
seems strange to use `network` and `svgParameters` instead of `n` and `s` the parameters of the provider factory here. Maybe it will be the same objects, but seems incorrect.